terraform/implementation/main.tf

```diff
@@ -19,16 +19,21 @@ locals {
     resource_group_name = var.resource_group_name,
     subscription_id     = var.subscription_id,
   }))
+}
 
-  services = toset([
-    "fhir-converter",
-    "ingestion",
-    "ingress",
-    "message-parser",
-    "validation",
-  ])
+variable "services_to_chart" {
+  type = map(string)
+  default = {
+    fhir-converter = "fhir-converter-chart",
+    ingestion      = "ingestion-chart",
+    ingress        = "ingress-chart",
+    message-parser = "message-parser-chart",
+    orchestration  = "orchestration",
```

Comment:
note: `orchestration-chart` didn't work because the name kept cutting off right at `orchestration-` and TF would complain that it was an invalid name. The solution here was to rename the chart at the Helm level, and create this mapping to track microservice name <=> chart name.